.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Feature request is not always a problem, could be a great idea that would enhance the tool.  We may want to steer away from the "problem" language and towards "enhancement", "feature", "idea", "improvement", etc.

Reply:
@drey-bigney, from your feedback, I rearranged the articulation of problem vs improvement, and tightened the questions some. thank you.